### DISCUSSION TITLE
Welcome to LBS Discussions!

### DISCUSSION THREAD

**Discussion by Rynstick** [Rynstick/LBS#1] [Announcements]
Discussion :

**Comment by dhinakaran1986** [comment]
Hi friend
   I am using Wincc Unified V17.
your library can't able to open.
can you send in Wincc Unified v17 library.
dhinakarany1986@gmail.com
Thanks

**Reply by Rynstick** [reply]
Currently lbs is only available in v18 I might take a look and downgrade it to v17 at some points

**Comment by dhinakaran1986** [comment]
Thanks friend
I am waiting for your reply

**Comment by iamradgo** [comment]
Hello @Rynstick ! first off, your library is awesome! We've been using it for sometime now and recently we found out that the https://www.lbs-unified.com/ is down for the documentation. Is there a way that it will be back up soon or get a pdf copy of the docu? Thanks!

**Reply by Rynstick** [reply]
Hi @iamradgo I'm very happy that my library can be usefull for you and your team ! has for the website i'll have to investigate for the reason of the shutdown . for the time being write to me at Andrew.massicotte@outlook.com and ill send you the markdown file tho understand that they are not has easy to read has the website !

**Reply by iamradgo** [reply]
Thank you sir! I just sent you an email.
